UM-USElab/gradecraft-development
# Reordered filenames to have view first, then role

### Status
**READY**

### Description
This PR renames all files and partials to use a _view_ and then _role_ pattern. It also renames "instructor" views to be more accurately "staff"

### Migrations
NO

### Steps to Test or Reproduce
Click through the assignments, challenges, badges index and show pages on both the staff and student side to confirm that all view partials are properly referenced. 

### Impacted Areas in Application
List general components of the application that this PR will affect:

* Challenges Index and Show
* Assignments Index and Show
* Badges Index and Show
* Grade Scheme Elements Index
